### PR TITLE
Update install.md

### PR DESCRIPTION
**Pull request by ezirmusitua**
## Explanation

Add zh-cn translation

## Related issue

N/A

## What type of PR is this

/kind documentation

## Proposed Changes

Translate i18n/zh/docusaurus-plugin-content-docs/current/start/install.md

### CODE REVIEW

**Comment by alabulei1** [comment]
Hello, I am a [code review bot](https://github.com/flows-network/github-pr-summary/) on [flows.network](https://flows.network/).

It could take a few minutes for me to analyze this PR. Relax, grab a cup of coffee and check back later. Thanks!

File: i18n/zh/docusaurus-plugin-content-docs/current/start/install.md [hunk 1]
```diff
@@ -2,130 +2,130 @@
...
-# Install and uninstall WasmEdge
+# 安装和卸载 WasmEdge
 
-This chapter will discuss ways to install and uninstall the WasmEdge Runtime on various OSes and platforms. We will cover how to install plug-ins to WasmEdge.
+本章将讨论在各种操作系统和平台上安装和卸载 WasmEdge 运行时的方法。同时也将介绍如何为 WasmEdge 安装插件。
 
 <!-- prettier-ignore -->
 :::note
-Docker Desktop 4.15+ already has WasmEdge bundled in its distribution binary. If you use Docker Desktop, you will not need to install WasmEdge separately. Check out [how to run WasmEdge apps in Docker Desktop.](build-and-run/docker_wasm.md)
+Docker Desktop 4.15+ 已经在其分发二进制文件中打包了 WasmEdge。如果你使用 Docker Desktop，则无需再单独安装 WasmEdge。详情请查看[如何在 Docker Desktop 中运行 WasmEdge 应用](https://chat.openai.com/c/build-and-run/docker_wasm.md)。
```
Comment (alabulei1) [inline i18n/zh/docusaurus-plugin-content-docs/current/start/install.md:11]: 这里的 openai 链接是不对的

File: i18n/zh/docusaurus-plugin-content-docs/current/start/install.md [hunk 2]
```diff
@@ -2,130 +2,130 @@
...
 
-- The `$HOME/.wasmedge/bin` directory contains the WasmEdge Runtime CLI executable files. You can copy and move them around on your file system.
+- `$HOME/.wasmedge/bin` 目录中包含 WasmEdge 运行时可执行文件。你可以在文件系统中复制和移动它们。
 
-  - The `wasmedge` tool is the standard WasmEdge runtime. You can use it from the CLI.
-    - Execute a WASM file: `wasmedge --dir .:. app.wasm`
-  - The `wasmedgec` tool is the ahead-of-time (AOT) compiler to compile a `.wasm` file into a native `.so` file (or `.dylib` on MacOS, `.dll` on Windows, or `.wasm` as the universal WASM format on all platforms). The `wasmedge` can then execute the output file.
+  - `wasmedge` 工具是标准的 WasmEdge 运行时。你可以通过命令行界面 (CLI) 使用它。
+    - 执行一个 WASM 文件：`wasmedge --dir .:. app.wasm`
+  - `wasmedgec` 工具是提前（AOT）编译器，用于将 `.wasm` 文件编译成本机的 `.so` 文件（在 MacOS 上则是 `.dylib`，在 Windows 上则是 `.dll`，或者在所有平台上通用的 `.wasm` 格式）。然后 `wasmedge` 可以执行输出的文件。
```
Comment (alabulei1) [inline i18n/zh/docusaurus-plugin-content-docs/current/start/install.md:102]: 这里应该是 `wasm compile`

File: i18n/zh/docusaurus-plugin-content-docs/current/start/install.md [hunk 3]
```diff
@@ -2,130 +2,130 @@
...
 
-    - Compile a WASM file into a AOT-compiled WASM: `wasmedgec app.wasm app.so`
-    - Execute the WASM in AOT mode: `wasmedge --dir .:. app.so`
+    - 将 WASM 文件编译为 AOT 编译的 WASM：`wasmedgec app.wasm app.so`
```
Comment (alabulei1) [inline i18n/zh/docusaurus-plugin-content-docs/current/start/install.md:104]: `wasmedgec` 应该是 `wasmedge compile`

**Review by alabulei1 — APPROVED** [review]

**Comment by alabulei1** [comment]
Hi  @ezirmusitua 

Please check out my comments. Thanks

File: i18n/zh/docusaurus-plugin-content-docs/current/start/install.md (hunk 2 above)
Comment (ezirmusitua) [reply i18n/zh/docusaurus-plugin-content-docs/current/start/install.md]: 不确定你说的是哪一段，我对比了[原文](https://github.com/WasmEdge/docs/blob/3991760780a454b76984418873df8d03768578ed/i18n/zh/docusaurus-plugin-content-docs/current/start/install.md?plain=1#L102)，没有看到相关的内容

File: i18n/zh/docusaurus-plugin-content-docs/current/start/install.md (hunk 3 above)
Comment (ezirmusitua) [reply i18n/zh/docusaurus-plugin-content-docs/current/start/install.md]: `wasmedgec` 应该还没有被弃用，在文档后面有[内容](https://github.com/WasmEdge/docs/blob/3991760780a454b76984418873df8d03768578ed/i18n/zh/docusaurus-plugin-content-docs/current/start/install.md?plain=1#L109C10-L109C10)说明了 `wasmedgec` 和 `wasmedge compile` 是等效的。

**Pull request merged**